src/index.ts

```diff
@@ -72,15 +75,24 @@ export class Resolver {
    * @returns {Promise<AccountId>}
    */
   public async resolveSLD(domain: string): Promise<string | undefined> {
+    if (await this._udResolver.isSupportedDomain(domain)) {
+      try {
+      const udResult = await this._udResolver.addr(domain, 'HBAR')
+      return udResult;
+      } catch (error) {
+        console.log(error);
```

Comment:
Can you also remove the console.log here please?

Reply:
Fixed